### ISSUE TITLE
NPE when selecting (empty, non src) folder in Package Explorer

### ISSUE THREAD

**Issue by hallvard** [hallvard/plantuml#132]
I created a test folder named puml in a Java project and got an NPE when it was selected:

java.lang.NullPointerException: Cannot invoke "org.eclipse.core.runtime.IPath.toString()" because the return value of "net.sourceforge.plantuml.eclipse.utils.WorkbenchEditorPartDiagramIntentProviderContext.getPath()" is null
	at net.sourceforge.plantuml.text.AbstractTextDiagramIntentProvider.getDiagramInfos(AbstractTextDiagramIntentProvider.java:53)
	at net.sourceforge.plantuml.text.AbstractDiagramIntentProvider.getDiagramInfos(AbstractDiagramIntentProvider.java:119)
	at net.sourceforge.plantuml.eclipse.views.AbstractDiagramSourceView.updateDiagramText(AbstractDiagramSourceView.java:517)

**Comment by iloveeclipse** [comment]
Just saw this NPE in recent 1.1.25 version with following stack:
```
java.lang.NullPointerException
	at net.sourceforge.plantuml.text.AbstractTextDiagramIntentProvider.getDiagramInfos(AbstractTextDiagramIntentProvider.java:53)
	at net.sourceforge.plantuml.text.AbstractDiagramIntentProvider.getDiagramInfos(AbstractDiagramIntentProvider.java:119)
	at net.sourceforge.plantuml.eclipse.views.AbstractDiagramSourceView.updateDiagramText(AbstractDiagramSourceView.java:517)
	at net.sourceforge.plantuml.eclipse.views.AbstractDiagramSourceView.updateDiagramText(AbstractDiagramSourceView.java:442)
	at net.sourceforge.plantuml.eclipse.views.AbstractDiagramSourceView$DiagramTextChangedListener.diagramChanged(AbstractDiagramSourceView.java:349)
	at net.sourceforge.plantuml.eclipse.views.AbstractDiagramSourceView$DiagramTextChangedListener.selectionChanged(AbstractDiagramSourceView.java:344)
	at org.eclipse.ui.internal.e4.compatibility.SelectionService.notifyListeners(SelectionService.java:266)
	at org.eclipse.ui.internal.e4.compatibility.SelectionService.notifyListeners(SelectionService.java:180)
	at org.eclipse.ui.internal.WorkbenchPage$E4PartListener.partActivated(WorkbenchPage.java:220)
	at org.eclipse.e4.ui.internal.workbench.PartServiceImpl$2.run(PartServiceImpl.java:249)
	at org.eclipse.core.runtime.SafeRunner.run(SafeRunner.java:45)
	at org.eclipse.e4.ui.internal.workbench.PartServiceImpl.firePartActivated(PartServiceImpl.java:246)
	at org.eclipse.e4.ui.internal.workbench.PartServiceImpl.activate(PartServiceImpl.java:770)
	at org.eclipse.e4.ui.internal.workbench.PartServiceImpl.activate(PartServiceImpl.java:679)
	at org.eclipse.e4.ui.internal.workbench.swt.AbstractPartRenderer.activate(AbstractPartRenderer.java:97)
```
@hallvard : any ideas when the new 1.1.26 version will be released?

**Comment by hallvard** [comment]
I've released 1.1.26 now, could you please try it? If it works I will announce and release on GitHub.

**Comment by jftz** [comment]
Thank you @hallvard! Just installed it. Working great!
